docs/deploy.md

```diff
@@ -548,3 +548,102 @@ This exposes a JSON-RPC interface to the Hardhat Network. Use the address, **htt
 
+## Verifying a Smart Contract
+
+Any deployed smart contract can be verified using the EON Explorer.  The verification process begins with submitting the smart contract's source code to EON Explorer. The EON Explorer compares the submitted source code with the deployed byte code. If there is a match, then the smart contract is verified successfully, whereby the EON Explorer generates an interface to call the methods of the contract.
+
+**Note:** It is highly recommended that you use the **Chrome web browser** to interact with the **EON Explorer** for optimized user experience.
+
+The major steps for verifying a smart contract are:
+
+* Deploy a smart contract to the Yuma Testnet network
+
+
+**Note:** See the [Build](https://eon.horizen.io/docs/build) section for details on compiling and deploying a smart contract.
+
+* Flatten the smart contract, if needed
+* Check for the deployed contract using the EON Explorer
+* Verify and publish the contract
+
+Make sure that you deploy your contract to the Yuma Testnet. If you are using a wallet, you must connect it to the Yuma Testnet network.
+
+Use the following parameters to connect to Yuma Testnet:
+
+```
+Network name:  Yuma Testnet 
+New RPC URL:  https://yuma-testnet.horizenlabs.io/ethv1 
+Chain ID: 1662 
+Currency symbol: ZEN
+```
+
+In this exercise, the [EON Explorer](https://yuma-explorer.horizen.io/) is connected to the Yuma Testnet. The smart contract was developed and deployed to the Yuma Testnet network. The verification process will be performed using the **Via flattened source code** option in the EON Explorer.
+
+### Flatten the Smart Contract
+
+Before using the EON Explorer for verification, make sure the Yuma Testnet network has the deployed smart contract you want to verify. You need to flatten the smart contract if the contract inherits from other contracts or if it uses a library. Flattening a smart contract refers to combining all Solidity code into one file instead of multiple source files, thereby having the imported code be embedded in the same file. 
+
+All smart contract development tools have a source code flattener. For example: 
+
+* **Truffle** - uses the [Truffle Flattener](https://www.npmjs.com/package/truffle-flattener)
+* **Hardhat** - uses the [Hardhat Flattener](https://hardhat.org/hardhat-runner/docs/advanced/flattening) 
+* The [POA Solidity Flattener](https://github.com/poanetwork/solidity-flattener) can be used to flatten a smart contract
+* **Remix** - provides a built-in flattener tool
+ 
+**Note:** In the **Remix Explorer**, select your smart contract file ( **.sol** ) to flatten, then right-click on the flattener tool. 
+
+### Check for the Deployed Contract
+
+In the [EON Explorer](https://yuma-explorer.horizen.io/), perform the following steps to begin the verification process:
+
+1. Once you have deployed your contract, your developer tool will display a contract address to check the pending transaction. Copy the contract address.
```

Comment:
"check the pending transaction" is not correct. if the contract address has been returned that means the transaction is confirmed. You could say instead: "Once you have deployed your contract, your developer tool will display the address the network has assigned your contract to. Copy that address."